packager/rbi_gen.cc

```diff
@@ -34,7 +34,7 @@ class Output final {
 public:
     template <typename... Args> void println(fmt::format_string<Args...> fmt, Args &&...args) {
         fmt::format_to(std::back_inserter(out), "{:{}}", "", this->indent * 2);
-        fmt::format_to(std::back_inserter(out), fmt, args...);
```

Comment:
`std::forward` is necessary to allow the use of formatters like `fmt::join` with `println`.